h/api/storage.py

```diff
@@ -44,6 +46,154 @@ def fetch_annotation(request, id):
     return elastic.Annotation.fetch(id)
 
 
+def _legacy_create_annotation_in_elasticsearch(request, data):
+    annotation = elastic.Annotation(data)
+    # FIXME: this should happen when indexing, not storing.
+    _prepare(request, annotation)
+    annotation.save()
+    return annotation
+
+
+def transform_links(document, claimant, created, updated):
```

Comment:
I added the `created` and `updated` since when we migrate the data over to Postgres we don't want to have the current timestamp as the value for document objects. I'm not sure if it makes sense to do this when we write annotations from Postgres directly from the views.